examples/helm-charts/cubestack/values.yaml

```diff
@@ -0,0 +1,810 @@
+## Override the name
+##
+nameOverride: ""
+
+## Provide a name to substitute for the full names of resources
+##
+fullnameOverride: ""
+
+##  Labels to add to all deployed objects
+##
+commonLabels: {}
+
+## Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+
+#### Cubejs ####
+################
+cubejs:
+  enabled: true
```

Comment:
I think there is no need to copy all values here. You can just keep the `xxx.enabled` values